### ISSUE TITLE
In 1.8.0 is not possible create the difference from two models

### ISSUE THREAD

**Issue by DavideAlidosi** [wkpark/sd-webui-model-mixer#131]
I am trying to make the difference between two SDXL models but with WebUI 1.8.0 it seems to be not possible.
This is my workflow:

1) I select Difference between base and current.
2) I load the two models
3) I enable without LoRAs.
4) I name the new LoRA.
5) I press save

In case it is a LyCORIS I get the following error:
`No lycoris module found No module named 'diffusers'`
In the case of a LoRA
`No scripts.kohya.* modules found. ERROR: No module named 'diffusers'`

On 1.7.0 I have no errors at all.

**Comment by DavideAlidosi** [comment]
I've found a fix, just need to copy system\python\Lib\site-packages\diffusers folder from 1.7.0 to 1.8.0 installation.
Probably just need to update this dependency into a new commit.
Bests.

**Comment by wkpark** [comment]
yes. diffusers needed to use Lycoris/LoRA extractor. but diffusers dependency is not included in the `requirements.txt` or `install.py` for some reason. (diffusers used by many other webui-extensions but the versions are not exactly matched so I did not include diffusers in the `requirements.txt` and this is undocumented.)
thanks for your reporting!
